gpkit/nomials/array.py

```diff
@@ -80,7 +79,7 @@ def latex(self, matwrap=True):
             return None
 
     def __hash__(self):
-        return hash(reduce(xor, map(hash, self.flat), 0))
+        return reduce(xor, map(hash, self.flat), 0)
```

Comment:
Nope, they'll be inequal and so won't get confused in the dict. Python dicts are friendly to stupid hash functions.